Source/Documentation/Manual/sound.rst

```diff
@@ -284,6 +284,33 @@ Trigger       Function
 Variable Triggers
 -----------------
 
+ORTS
+^^^^
+
+The sound objects attached to a vehicle (wagon or loco) can respond in volume and frequency to changes in the vehicle's properties.
+There are 7 properties:		
+- distance squared from a sound source (m^2)
```

Comment:
```suggestion
- distance squared from a sound source (m\ :sup:`2`)
```
A superscript would look a little nicer here.

Reply:
Hi Ryan,

Taken me a while to get round to this.

Done now. Please approve.

Chris